DEV-27 contents (#128)
IHE/sdpi-fhir: SDPi_Supplement/asciidoc/volume2/dev-27/tf2-dev-27.adoc

:var_transaction_id: DEV-27

:var_label_dev_27_message_subscribe: Subscribe
:var_label_dev_27_message_subscribe_response: SubscribeResponse
:var_label_dev_27_message_notification: Notification
:var_label_dev_27_message_renew: Renew
:var_label_dev_27_message_renew_response: RenewResponse
:var_label_dev_27_message_unsubscribe: Unsubscribe
:var_label_dev_27_message_unsubscribe_response: UnsubscribeResponse
:var_label_dev_27_message_subscription_end: SubscriptionEnd

[#vol2_clause_dev_27]
=== Manage BICEPS Subscription [{var_transaction_id}]

==== Scope

// [sdpi_transaction#transaction_dev_27,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_27_probe_sequence,messages='message_announce_network_presence']

This transaction is used to establish a publish-subscribe session between a <<actor_somds_provider>>, acting as the event source, and a <<actor_somds_consumer>>, acting as the event sink.

==== Actor Roles

.Actor Roles {var_transaction_id}
[cols="1,2"]
|===
|Actor |Roles

|<<actor_somds_consumer>>

> |Initiates communication by first subscribing to a <<actor_somds_provider>> by sending a <<vol2_clause_dev_27_message_subscribe, {var_label_dev_27_message_subscribe}>> message. While the subscription is running, the subscription is kept alive by sending <<vol2_clause_dev_27_message_renew, {var_label_dev_27_message_renew}>> messages and <<vol2_clause_dev_27_message_notification, {var_label_dev_27_message_notification}>> messages are being received.

Reviewer: ```suggestion
|Initiates communication by first subscribing to a <<actor_somds_provider>> by sending a <<vol2_clause_dev_27_message_subscribe, {var_label_dev_27_message_subscribe}>> message. While the subscription is running, the subscription is kept alive by sending <<vol2_clause_dev_27_message_renew, {var_label_dev_27_message_renew}>> messages and receiving <<vol2_clause_dev_27_message_notification, {var_label_dev_27_message_notification}>> messages as response.

```

Author: As agreed:

```suggestion
|Initiates communication by first subscribing to a <<actor_somds_provider>> by sending a <<vol2_clause_dev_27_message_subscribe, {var_label_dev_27_message_subscribe}>> message. While the subscription is running, the subscription is kept alive by sending <<vol2_clause_dev_27_message_renew, {var_label_dev_27_message_renew}>> messages and receiving <<vol2_clause_dev_27_message_notification, {var_label_dev_27_message_notification}>> messages.
```
